### ISSUE TITLE
input variable for list not showing

### ISSUE THREAD

**Issue by oba11** [segmentio/terraform-docs#47]
Noticed that if the default value of a variable is list, it doesnt show the value.

e.g

```
variable "instance_type" {
  description = "Instance types to be launched"
  type        = "list"
  default     =  ["c5.large"]
}
```

What the doc generates is below

| Name | Description | Type | Default | Required |
|------|-------------|:----:|:-----:|:-----:|
| instance_type| Instance types to be launched by the spot compute environment | list | `<list>` | no |

Thanks

**Comment by metmajer** [comment]
@oba11 I have just added a PR for this https://github.com/segmentio/terraform-docs/pull/53

**Comment by metmajer** [comment]
@oba11 The request has been resolved with #53.

**Comment by oba11** [comment]
Awesome, it works perfectly, thanks alot @metmajer 🎉
